Vagrantfile.erb

```diff
@@ -0,0 +1,58 @@
+Vagrant.configure("2") do |c|
+  c.vm.box = "<%= config[:box] %>"
+  c.vm.box_url = "<%= config[:box_url] %>"
```

Comment:
I had to comment this line otherwise, a fresh download of the box was not possible.

Reply:
Alright. It still works for me if I remove that line, too. So it's gone!